### ISSUE TITLE
Webpack 2 sassResources has wrong type.

### ISSUE THREAD

**Issue by petermezei** [shakacode/sass-resources-loader#15]
Hi Guys,

after updating my webpack to webpack 2.1.0-beta27, I got the error message, sassResources has wring type.

My webpack config part:

```javascript
plugins: [
    new webpack.LoaderOptionsPlugin({
      options: {
        sassResources: [
          './node_modules/bootstrap-sass/assets/stylesheets/bootstrap/_variables.scss',
          './node_modules/bootstrap-sass/assets/stylesheets/bootstrap/mixins/*.scss'
        ]
      }
    }),
   ...
]
```

Print screen:

![image](https://cloud.githubusercontent.com/assets/18461114/21073221/ddf67874-bed8-11e6-9b8e-c54a4f8ad5db.png)

Could you help me please?

**Comment by petermezei** [comment]
Bypass solution.

Removed sass-resources.
Added sass-loader option:

```javascript
new webpack.LoaderOptionsPlugin({
      options: {
        sassResources: [
          //'demo.scss'
        ],
        sassLoader: {
          includePaths: [path.resolve(__dirname, "./src/assets")]
        }
      }
    }),
   ...
]
```

created _common.scss in /src/assets

```scss
@import "./node_modules/bootstrap-sass/assets/stylesheets/bootstrap/_variables.scss";
@import "./node_modules/bootstrap-sass/assets/stylesheets/bootstrap/mixins";
```

Added @import "common" into the needed sass files.

**Comment by justin808** [comment]
@petermezei Is there an issue here? These are 2 very different npm modules.

**Comment by petermezei** [comment]
hi @justin808 do you mean webpack 2 does not compatible with sass-resources-loader 1.1?

**Comment by justin808** [comment]
@alexfedoseev, @robwise, We're using v2 with this package, correct?

@petermezei it should be working with v2!

**Comment by robwise** [comment]
@justin808 Yep, I'm using webpack v2 with sass-resources-loader 1.1 on two different projects ATM.

**Comment by alexfedoseev** [comment]
> @alexfedoseev, @robwise, We're using v2 with this package, correct?

`sass-resources-loader` is broken started from `webpack@2.1.0-beta.23`. Since this version webpack doesn't allow custom properties on config object (eg`sassResources`).

**Comment by robwise** [comment]
Oh woops, that's right I forgot

**Comment by Nipol** [comment]
Hello. I saw the following error message:

```
Path.js: 7
    Throw new TypeError ('Path must be a string. Received' + inspect (path));
    ^

TypeError: Path must be a string. Received undefined
    At assertPath (path.js: 7: 11)
    At Object.relative (path.js: 1226: 5)
    At Object.onRender (/Users/asdf/workspace/webpack_example/node_modules/sass-loader/index.js:282:42)
    At Object. <Anonymous> (/Users/asdf/workspace/webpack_example/node_modules/async/dist/async.js:2234:31)
    At apply (/Users/asdf/workspace/webpack_example/node_modules/async/dist/async.js:20:25)
    At Object. <Anonymous> (/Users/asdf/workspace/webpack_example/node_modules/async/dist/async.js:56:12)
    At Object.callback (/Users/asdf/workspace/webpack_example/node_modules/async/dist/async.js:840:16)
    At options.success (/Users/asdf/workspace/webpack_example/node_modules/node-sass/lib/index.js:303:32)
```

```
New webpack.LoaderOptionsPlugin ({
      Options: {
        SassResources: [
          '_common.scss'
        ],
        SassLoader: {
          IncludePaths: [path.resolve (__ dirname, "./app/assets")]
        }
      }
    }),
```

Do you know where this error is? I have no idea.

I am using 'nodejs 6.9.2' and 'webpack@2.1.0-beta.27'.

plz help.

**Comment by Razinsky** [comment]
I don't know if it can help but I have the same issue as @petermezei. I see from its screenshot that he is using Typescript and so am I... Maybe there's something here?

**Comment by justin808** [comment]
@Nipol @Razinsky @petermezei Any chance of helping us with a review: https://reviewable.io/reviews/shakacode/sass-resources-loader/16#-

and maybe trying this out?

**Comment by justin808** [comment]
I pushed up a beta: 1.2.0-beta.1

**Comment by vanyadymousky** [comment]
Tested with webpack 2.2.1, sass-resources-loader v 1.0.2 works perfectly, 1.2.0-beta.2 doesn't work at all.

**Comment by alexfedoseev** [comment]
@vanyadymousky If you want to get help w/ the issue, provide more details, e.g.:

1. error stack or screenshot of the error
2. repo w/ your configuration to reproduce the issue

**Comment by alexfedoseev** [comment]
Fixed in `1.2.0`.

@vanyadymousky If you still experience problems w/ `1.2.0`, feel free to create new issue. Please, don't forget to provide details I mentioned above.
